packages-content-model/roosterjs-content-model-api/test/modelApi/block/setModelDirectionTest.ts

```diff
@@ -2,15 +2,25 @@ import { ContentModelDocument } from 'roosterjs-content-model-types';
 import { setModelDirection } from '../../../lib/modelApi/block/setModelDirection';
 
 describe('setModelDirection', () => {
+    const width = '3px';
+    const style = 'double';
+    const color = '#AABBCC';
+    const testBorderString = `${width} ${style} ${color}`;
+
     function runTest(
         model: ContentModelDocument,
         direction: 'ltr' | 'rtl',
         expectedModel: ContentModelDocument,
-        expectedResult: boolean
+        expectedResult: boolean,
+        tableTest?: boolean
     ) {
         const result = setModelDirection(model, direction);
 
         expect(result).toBe(expectedResult);
+
+        if (tableTest) {
+            model.blocks[0]['dataset'] = {};
+        }
```

Comment:
please fix build. If it is expected to be "any" type, you can add a type declaration like:
model.block[0]['dataset'] = {} as any;